plugins/repository-gcs/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -43,4 +43,5 @@ grant {
 
   // gcs client set Authenticator for proxy username/password
   permission java.net.NetPermission "setDefaultAuthenticator";
+  permission java.io.FilePermission "/sys/class/dmi/id/product_name", "read";
```

Comment:
This line and the below change are due to the change in the new version of the jar, see the [gradle check result](https://build.ci.opensearch.org/job/gradle-check/50289/testReport/junit/org.opensearch.repositories.gcs/GoogleCloudStorageServiceTests/testApplicationDefaultCredentialsWhenNoSettingProvided/).

Reply:
As far as I can tell, we do not open up access to any resource that is outside of `config` folder.

Reply:
Could you please revert this change and add to the build.gradle:

```
test {
  environment 'NO_GCE_CHECK', 'true'
}
```